node/src/config.rs

```diff
@@ -35,22 +40,26 @@ mod tests {
                 CONFIG_FILENAME,
                 r#"
                     [block_producer]
-                    store_url = "http://store:8000"
+                    store_url = "http://store:82"
 
                     [block_producer.endpoint]
-                    host = "127.0.0.1"
-                    port = 8080
+                    host = "0.0.0.0"
+                    port = 81
```

Comment:
These changes are not necessary. I added them mostly to show that the services should run on public interfaces / ip addresses, whereas the control plane should only be available for the localhost.

Reply:
Yep - that's fine. One comment: only the RPC component should be run on "public" interfaces / IPs. "Public" here means exposed to the general public. I think in your context, public meant public to the machine, rather than the network, right?